src/main/webapp/dev/js/common/questionRank.es6

```diff
@@ -9,7 +9,7 @@ function addRankOption(questionNum) {
             parseInt($(`#${ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}-${questionNum}`).val(), 10);
 
     $(`
-    <div id="rankOptionRow-${curNumberOfChoiceCreated}-${questionNum}">
+    <div id="rankOptionRow-${curNumberOfChoiceCreated}-${questionNum}" class="margin-bottom-7px">
```

Comment:
Place `class` before `id` for consistency with [`feedbackQuestionRankEditFormOptionFragment.html`](https://github.com/TEAMMATES/teammates/blob/e1052601153adaa0f64f4b231d46a460cc931373/src/main/resources/feedbackQuestionRankEditFormOptionFragment.html#L1) and the test files?